.bumpversion.cfg

```diff
@@ -3,7 +3,7 @@ commit = True
 tag = True
 sign_tags = True
 message = build: Version {new_version}
-current_version = 2.4.0
+current_version = 2.5.0
```

Comment:
Please update the version compatibility matrix in the README (currently it says Quince must use 2.4).

Reply:
Sure, what should it say, `>=2.4, <2.6` ?

Reply:
Ack, done.